packages/react/src/components/pickers/TagPicker/TagPicker.types.ts

```diff
@@ -47,6 +47,9 @@ export interface ITagItemProps extends IPickerItemProps<ITag> {
 
   /** Theme provided by High-Order Component. */
   theme?: ITheme;
+
+  /** data-id attribute for the remove button */
+  'remove-button-data-id'?: string;
```

Comment:
Alternatively, we can pass a whole object. Something like

```
type RemoveButtonProps = {
  'data-id'?: string
}

...

removeButtonProps?: RemoveButtonProps 
```

Reply:
I'm in favor of this as it's a more general solution. Can you make the update to this PR?

Reply:
will do 👍

Reply:
hello @spmonahan I have changed it from string prop to an object